tools/actions/create-release-proposal.sh

```diff
@@ -31,8 +31,9 @@ HEAD_SHA="$(git rev-parse HEAD^)"
 
 TITLE="$(git log -1 --format=%s)"
 
-# Use a temporary file for the PR body
-TEMP_BODY="$(awk "/## ${RELEASE_DATE}/,/^<a id=/{ if (!/^<a id=/) print }" "doc/changelogs/CHANGELOG_V${RELEASE_LINE}.md")"
+TEMP_BODY="$(awk -v MAX_BODY_LENGTH="65536" \
+    "/^## ${RELEASE_DATE}/,/^<a id=/{ if (/^<a id=/) {exit;} if (output_length += length(\$0) > MAX_BODY_LENGTH) {exit 1;} print }" \
```

Comment:
looks like I forgot a pair of parentheses, sorry for that
```suggestion
    "/^## ${RELEASE_DATE}/,/^<a id=/{ if (/^<a id=/) {exit;} if ((output_length += length(\$0)) > MAX_BODY_LENGTH) {exit 1;} print }" \
```

Reply:
no worries. thx